dice_roller.py

```diff
@@ -0,0 +1,93 @@
+import discord
+from discord.ext import commands
+import random
+import operator as op
+import time
+
+intents = discord.Intents.default()
+intents.message_content = True
+intents.presences = True
+
+# Invite Link: https://discord.com/oauth2/authorize?client_id=1147014445474787409&permissions=68164851858518&scope=bot
+
+# Setup bot with command prefix '/'
+diceRollerBot = commands.Bot(command_prefix='/', intents=intents)
+
+# Define set to contain valid types of checks for command parameter
+CHECKTYPESET = {"athletics", "acrobatics", "sleightofhand",
+                "stealth", "arcana", "history", "investigation",
+                "nature", "religion", "animalhandling",
+                "insight", "medicine", "perception", "survival",
+                "deception",  "intimidation", "performance",
+                "persuasion"}
+
+# Bot event: When bot is ready send message
+@diceRollerBot.event
+async def on_ready():
+    print(f'{diceRollerBot.user.name} ~Ready to perform checks ...')
+
+# Ability Check processing to handle checks that have more than one string
+# I.e. Ability Checks that are seperated with whitespace characters
+async def processArgs(*args):
+    temp: str = ""
+    # Traverse through the arguments
+    for i in args:
+        temp = temp + " " + i
+
+    print(f'CONSOLE: Concatenated argument is: {temp}')
+    return temp
```

Comment:
Replace the for loop with the join method. If you want space in between arguments then do the following:

```suggestion
    temp = " ".join(args)
    print(f'CONSOLE: Concatenated argument is: {temp}')
    return temp
```

Reply:
👍🏽